### PR TITLE
[kokoro] Add two different simulator runs

### PR DESCRIPTION
**Pull request by jverkoey**
We may eventually want to turn this into a parallelized matrix of jobs, but for now our build times are fast enough that we may be able to get away with a sequence of jobs.

Part of https://github.com/material-components/material-components-ios/issues/2461.

### CODE REVIEW

File: .kokoro [hunk 1]
```diff
@@ -71,11 +71,25 @@ run_bazel() {
     verbosity_args="-v"
   fi
 
+  # The iPad Pro 12.9 was chosen because it is the largest resolution we can test on.
+  # 9.3 was chosen to round out the range of iOS versions we are testing on.
```
Comment (romoore) [inline .kokoro:75]: Leave a TODO to update once iOS 10 (11?) support is available?

**Review by romoore — APPROVED** [review]
Does it make sense to build for both 32-bit and 64-bit architectures for a 32-bit simulator (iPhone 5)?  Same question goes for the IPad pro, I guess.

**Comment by romoore** [comment]
Looks like an issue parsing the iPad Pro target?

    ./.kokoro-ios-runner/bazel.sh test //components/... --min-xcode-version 8.2.0 -v --ios_minimum_os=8.0 --ios_multi_cpus=i386,x86_64 '--ios_simulator_device=iPad Pro (12.9-inch)' --ios_simulator_version=9.3
    ERROR: no such target '//:Pro': target 'Pro' not declared in package '' 

We may need to wrap it in additional quotes.

**Comment by jverkoey** [comment]
Yep - there's a bug in the kokoro bazel script as well. Will send out a patch now.

**Comment by willlarche** [comment]
I'm a little confused. Does this not give us all 4 of the configurations we have in Travis?

**Comment by jverkoey** [comment]
Xcode 9 still hasn't been deployed to kokoro unfortunately (b/67373848), so we can't add the Xcode 9 targets that we presently have in Travis.

**Comment by jverkoey** [comment]
This PR is unfortunately blocked for the foreseeable future because it appears that bazel doesn't make use of the `--ios_simulator_device` argument. I believe this is the case because `--ios_simulator_device='iPhjksdfljadslkfasone 5bsdf'` results in a successful test run.

**Comment by willlarche** [comment]
Since we can't do Xcode 9, let's do ios 8, 9, 10 on xcode 8 with:

smallest 32-bit iphone
largest 64-bit iphone
largest iPad.

**Review by willlarche — CHANGES_REQUESTED** [review]
See note about 3 requested simulator configs.

**Comment by romoore** [comment]
Can we close this PR for now and just track it in an issue/feature request for Bazel?

**Comment by jverkoey** [comment]
Yeah I think that's reasonable.

**Pull request closed without merging**